src/allencell_ml_segmenter/core/file_input_widget.py

```diff
@@ -51,9 +51,10 @@ def __init__(
         model: FileInputModel,
         viewer: IViewer,
         service: ModelFileService,
+        include_channel_selection: bool = True,
     ):
         super().__init__()
-
+        self._include_channel_selection: bool = include_channel_selection
```

Comment:
Are these changes generalizing the fileInputWidget to not always deal with channels (for this use case)?

Reply:
We don't need to deal with channels because segmentations are already single channel, right? Is there any harm in just having the channel selection option there but with only one option?